src/main/Config/AppConfig.java

```diff
@@ -105,14 +113,58 @@ public static Javalin appFactory(DeploymentLevel deploymentLevel) {
     app.post("/get-all-activities", activityController.findMyActivities);
 
     /* --------------- PRODUCTION API --------------- */
+
+    // TODO use Access manager for this instead https://javalin.io/documentation#access-manager
+    app.before("/organizations", ctx -> {
```

Comment:
do you want to put this stuff in the controller and just throw when validation fails? Or is this temporary until Access Manager is added

Reply:
Is there a way to add code in the controller that is run before every handler it contains?